### PR TITLE
chore: add license to poetry

### PR DESCRIPTION
**Pull request by centenor**
Add open source license to pyproject.toml

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -3,6 +3,7 @@ name = "secureli"
 version = "0.2.14"
 description = "Secure Project Manager"
 authors = ["Tristan Leonard <tristanl@slalom.com>"]
```
Comment (centenor) [inline pyproject.toml:5]: Do we want to change the author?

**Comment by centenor** [comment]
Seems the build pipeline is currently in a broken state

File: pyproject.toml (hunk 1 above)
Comment (calebtonn) [reply pyproject.toml:5]: I technically didn't author it, but I am ultimately responsible so let's switch it to me. caleb.tonn@slalom.com

**Review by calebtonn — COMMENTED** [review]
It looks like the build is failing due to the name of the branch

**Pull request closed without merging**